main/getting-started/development-cycle.md

```diff
@@ -1,66 +0,0 @@
-# Development Cycle 
-
-Here's what you have to do when you make code changes in various parts of the system, including your contract code, so that the Agoric system reflects your changes.
-
-Different changes have different impacts. The table below is ordered so that your changes with the most impact are at the top in the first row. To use the table, go down the first column until you see a type of change you made.
-
-Then go to that row's second column and do the action specified there. You then need to go down the second column and do every action specified in the second column down to and including the last row's second column. In other words, if you do the action in row 3's second column, you must also do the action in row 4's second column, then row 5's second column, and so on until the bottom row.
-
-You don’t have to do any of the second column actions in rows above where you started.
-
-<table>
-<tbody>
-<tr>
-<td colspan="3"><b>How to use this table:</b><br><br>
-<ol><li><b>Go down the first column until you encounter the type of change you made.</b></li>
-<li><b>Go to that row’s second column and do the action specified there (if you want to know more about what and why you’re doing, see the row’s third column).</b></li>
-<li><b>Go down to the next row and do the action in its second column.</b></li>
-<li><b>Repeat 3. until you reach the bottom of the table and do the action in the bottom row’s second column.</b></li>
-</ol>
-For example, let’s say you changed code in <code>agoric-sdk</code>. Going down the first column, you find it and look at the action in the second column of that row. It says you need to run <code>agoric start --reset</code>. Then go to the next row and its second column to find out you need to run <code>agoric deploy contract/deploy.js</code> Continue doing the action specified in each following row’s second column until the “refresh your browser” instruction in the bottom row.
-</td>
-</tr>
-<tr>
-<th><b>If you...</b></td></th>
-<th><b>Then you must...</b></td></th>
-<th><b>Notes</b></td></th>
-</tr>
-<tr>
-<td>Change which npm packages are used in <code>agoric-sdk</code></td>
-<td> In your <code>agoric-sdk</code> directory, run <code>yarn install</code>
-<td> Updates the <code>node_modules</code> to match the current package declarations. </td>
-</tr>
-<tr>
-<td>Change packages in the <code>packages/</code> directory. For example, <code>SwingSet</code>, <code>eventual-send</code>, <code>default-evaluate-options</code>, <code>captp</code>, or <code>evaluate</code></td>
```

Comment:
I looked over the text in this file to see if any of it should be preserved... no, it's almost all stuff for developers of the SDK, not uses of the SDK.

Reply:
docs for developers of the SDK go in https://github.com/Agoric/agoric-sdk/blob/master/CONTRIBUTING.md